virocon/distributions.py

```diff
@@ -411,12 +677,51 @@ def _fit_lsq(self, data):
         
 class ExponentiatedWeibullDistribution(Distribution):
     """
-    An exponentiated Weibull distribution.
+    An exponentiated Weibull distribution as used by Haselsteiner et al. (2019)
```

Comment:
The first line should be just one sentence. 
"""
An exponentiated Weibull distribution. 

The parametrization used is the same as described by by Haselsteiner et al. (2019) [1]_. 
The distributions cumulative distribution function is given by: ...

"""
Is there no pdf given in the paper? It is not absolutely necessary, but I think it would be nice if we would always use the pdf to describe the distribution.

Reply:
Unfortunately a pdf is not indicated.